GoogleForCreators/web-stories-wp
# Story Locking: Fix crash if there is no user avatar

## Context



## Summary

If avatars are disables with a lock post the dashboard errors.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Login as author, create story and save draft. 
2. In another browser, login as an admin. 
3. Disable Avatars, Settings > Discussion > Show avatar
4. Go to dashboard.
5. Should see lock icon and not avatar. 

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8593
Fixes #8588
